bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -767,6 +813,52 @@ contract("Bridge NFT", async function(accounts) {
             await assertTokenHasBeenClaimed(this.sideNFTBridge.address, anotherAccount, sideToken)
           });
 
+      it("should claim with the fallback function", async function () {
+        await assertTokenIsLockedInBridge(this.NFTBridge.address, anotherAccount, this.token);
+        const receipt = await this.sideNFTBridge.claimFallback(
+          {
+            to: anotherAccount, from: anAccount, tokenId: tokenId, tokenAddress: tokenAddress,
+            blockHash: blockHash, transactionHash: transactionHash, logIndex: logIndex
+          },
+          {from: anAccount}
+        );
+
+        truffleAssert.eventEmitted(receipt, claimedNFTTokenEventType, (event) => {
+          return (
+            event._transactionHash === transactionHash &&
+            event._originalTokenAddress === tokenAddress &&
+            event._to === anotherAccount &&
+            event._sender === anAccount &&
+            event._tokenId.eq(new BN(tokenId)) &&
+            event._blockHash === blockHash &&
+            event._logIndex.eq(new BN(logIndex)) &&
+            event._receiver === anAccount
+          );
+        });
+      });
+
+      it("should check if it was claimed without claiming", async function () {
+        const transactionDataHash = await this.NFTBridge.getTransactionDataHash(anotherAccount, anAccount, tokenId, tokenAddress, blockHash, transactionHash, logIndex);
+        const result = await this.NFTBridge.hasBeenClaimed(transactionDataHash);
+        assert(!result, "The token should not been claimed yet");
+      });
+
+      it("should check if it was claimed successfully", async function () {

```

Comment:
this test is the opposite of the previous one (it tests that the negation of the assertion of the previous test is falsy), I think we should remove one of them)

Reply:
removed the first one at https://github.com/rsksmart/tokenbridge/pull/268/commits/9c4e1c7b64b7379ff804bb3f768eed9c96a7461b